apps/web/src/components/activity/helpers.ts

```diff
@@ -59,3 +73,11 @@ export const getLogoByType = (
 
   return null;
 };
+
+export const mappedWebhookStatuses = {
+  sent: { label: 'Sent', icon: Sent, status: ['processed'] },
+  received: { label: 'Received', icon: Received, status: ['delivered'] },
+  read: { label: 'Read', icon: Read, status: ['open'] },
```

Comment:
```suggestion
  read: { label: 'Read', icon: Read, status: ['read'] },
```
Or do we want to assumer that if `open` has been read too?
I haven't checked though if SMS and email return in the webhooks a read status.

Reply:
Not sure how `read` and `open` are different,
Also for now I just added the status provided by sendgrid
As the other providers start supporting webhook, we will have to update the relevant `status` arrays with the status provided by webhook